src/main/java/core/basesyntax/models/Square.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax.models;
+
+import core.basesyntax.Figure;
+
+public class Square implements Figure {
+    private final String color;
+    private final double side;
+
+    public Square(String color, double side) {
+        this.color = color;
+        this.side = side;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Color: " + color
+                + "Type: Square "
+                + "Side: " + side);

```

Comment:
The 'Type: Square' part of the message in `draw()` is not necessary as we already mention 'figure: square' at the beginning of the message.